arclight/app/helpers/application_helper.rb

```diff
@@ -4,4 +4,16 @@ module ApplicationHelper
   def additional_locale_routing_scopes
     [ blacklight, arclight_engine ]
   end
+
+  def repository_location_to_city(location_str)
```

Comment:
Would love a very small test for this. [Rspec docs for helpers](https://rspec.info/features/8-0/rspec-rails/helper-specs/helper-spec/) is pretty good